pages/linux/nordvpn.md

```diff
@@ -0,0 +1,24 @@
+# nordvpn
+
+> Commandline interface to NordVPN.
+> NordVPN's site: <https://nordvpn.com/>.
+
+- Login to your NordVPN account:
+
+`nordvpn login`
+
+- Display the connection status:
+
+`nordvpn status`
+
+- Connect to the nearest server:
+
+`nordvpn connect`
+
+- List countries:
+
+`nordvpn countries`
+
+- Connect to a specific country:
```

Comment:
Is there maybe a more technical expression for this? Maybe something like `Connect to the NordVPN server from a specific country:`